src/logic/calculate.js

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.